include/retdec/llvmir2hll/obtainer/call_info_obtainer.h

```diff
@@ -252,7 +252,7 @@ class CallInfoObtainer: public SharableFromThis<CallInfoObtainer>,
 
 protected:
 	/// Set of sets of functions.
```

Comment:
Is this description correct? I am referring to _"Set of sets"_. Technically, it is no longer a set. However, from a high-level point of view, I can understand the the description might be OK. What is your take on this?

We should also maybe consider adding a comment why a vector is a better choice here than a set.

Reply:
Let's just go with vector.
I've fixed this comment and added a comment to the computeSCCs explaining why we use it.